packages/web-components/src/dropdown/README.md

```diff
@@ -0,0 +1,175 @@
+# Dropdown
+
+> A dropdown is a user interface component that presents a collapsible list of selectable options to users.
+
+<br />
+
+## **Design Spec**
+
+[Link to Dropdown Design Spec in Figma](https://www.figma.com/file/pI5m9e9hsNkl1odWzxCxNd/Dropdown?node-id=1319-163&t=Hfpd1nwRmumsVKtL-0)
+
+<br />
+
+## **Engineering Spec**
+
+Fluent WC3 DroFluent WC3 Dropdown is a form associated component that extends from the FAST Select [FAST Select](https://explore.fast.design/components/fast-select) and is intended to be as close to the Fluent UI React 9 Dropdown implementation as possible. However, due to the nature of web components there will not be 100% parity between the two.
+
+<br />
+
+## **Class:** `Dropdown`
+
+<br />
+
+### **Component Name:** `fluent-dropdown`
+
+### Superclass
+
+| Name                   | Module                                                                                                                                                  | Package                      |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- |
+| `FormAssociatedSelect` | [select.form-associated.js](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/select/select.form-associated.ts) | `@microsoft/fast-foundation` |
+
+<br />
+
+### Static Fields
+
+| Name                  | Privacy | Description                                         | Inherited Form                                                                                                             |
+| --------------------- | ------- | --------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `slottedOptionFilter` |         | A static filter to include only selectable options. | [FASTListbox](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/listbox/README.md) |
+
+<br />
+
+### **Fields**
+
+| Name              | Privacy | Type                                                                                                             | Default   | Description                                          |
+| ----------------- | ------- | ---------------------------------------------------------------------------------------------------------------- | --------- | ---------------------------------------------------- |
+| `appearance`      | public  | `{ outline: 'outline', underline: 'underline', filledDarker: 'filled-darker', filledLighter: 'filled-lighter' }` | `outline` | Sets style variations.                               |
```

Comment:
Assume these appearances come from react fluent, `filled-lighter` and `filled-darker` semantics mean the same in both light and dark mode?

Reply:
This comes from design spec, the lighter variant is lighter in both light and dark theme 🤷‍♂️